Hestia/app/src/main/java/com/rugged/application/hestia/PeripheralListFragment.java

```diff
@@ -0,0 +1,103 @@
+package com.rugged.application.hestia;
```

Comment:
This class will specially benefit from cleanup and documentation. Unpleasant to parse otherwise.